linux_params.h

```diff
@@ -110,7 +110,15 @@ struct linux_params {
     uint32_t initrd_addr_max;           /* 0x22c */
     uint32_t kernel_alignment;          /* 0x230 */
     uint8_t relocatable_kernel;         /* 0x234 */
-    uint8_t reserved13[0x2b];           /* 0x235 */
```

Comment:
Not sure where the various `reserved` field names come from (probably already in the original `cbfstool` sources). But `reserved12_5` can become `reserved13` now. Then the numbering is consecutive once more.